go.mod

```diff
@@ -86,6 +85,7 @@ require (
 	go.opentelemetry.io/otel v1.35.0 // indirect
 	go.opentelemetry.io/otel/metric v1.35.0 // indirect
 	go.opentelemetry.io/otel/trace v1.35.0 // indirect
+	go.uber.org/automaxprocs v1.6.0 // indirect
```

Comment:
I was wondering why it's still a dependency.
Ginkgo depends on it, so it should not appear in our images anymore